contracts/Penalizer.sol

```diff
@@ -15,6 +15,8 @@ import "./interfaces/IPenalizer.sol";
 contract Penalizer is IPenalizer, Ownable {
 
     string public override versionPenalizer = "2.0.1+enveloping.penalizer.ipenalizer";
+    // bytes4(keccak256("penalize(address,address)"))
+    bytes4 private constant PENALIZE_SELECTOR = 0xebcd31ac;
```

Comment:
_love_ this from a programmatic point of view. 

however, would an in-line value reduce costs even further?

Reply:
I have just tried your suggestion and I hadn't observed any gas improvement. For the sake of readability, I would define it here, WDYT?

Reply:
yes, better to leave it as it is :+1: